db/initdb/dump.sql

```diff
@@ -302,37 +302,75 @@ ALTER TABLE public.craft_data_seq OWNER TO postgres;
 CREATE TABLE public.game_accounts (
     id bigint NOT NULL,
     dbaccount_id bigint,
+    lastonline bigint,
+    flags integer,
     banner bytea,
     uiprefs bytea,
     uisettings bytea,
     seentutorials bytea,
     bossprogress bytea,
     stashicons bytea,
-    lastonline bigint,
-    flags integer,
     paragonlevel integer,
     paragonlevelhardcore integer,
     experience integer,
     experiencehardcore integer,
     lastplayedhero_id bigint,
     gold bigint,
-    platinum integer,
     hardcoregold bigint,
+    platinum integer,
+    hardplatinum integer,
     rmtcurrency bigint,
+    hardrmtcurrency bigint,
     bloodshards integer,
     hardcorebloodshards integer,
     stashsize integer,
     hardcorestashsize integer,
     seasonstashsize integer,
+    hardseasonstashsize integer,
     eliteskilled bigint,
     totalkilled bigint,
     totalgold bigint,
+    hardtotalgold bigint,
     totalbloodshards integer,
+    hardtotalbloodshards integer,
     totalbounties integer DEFAULT 0 NOT NULL,
     totalbountieshardcore integer DEFAULT 0 NOT NULL,
     pvptotalkilled bigint,
+    hardpvptotalkilled bigint,
     pvptotalwins bigint,
-    pvptotalgold bigint
+    hardpvptotalwins bigint,
+    pvptotalgold bigint,
+    hardpvptotalgold bigint,
+    craftitem1 integer,
```

Comment:
what about seasonal craft items? By the way, I think it's better to extract consumables into a separate table like: `accountId | isSeasonal | isHardcore | craftitem1 | ... | craftitemN`